docs/index.md

```diff
@@ -0,0 +1,31 @@
+# Sweater Comb
+
+The Sweater Comb project is an executable manifestation of Snyk's API standards.
+
+Built with and powered by [Optic CI](https://www.useoptic.com/docs/) with 💜.
+
+# API Principles and Guidelines
+
+## [API Program](principles/api_program.md)
```

Comment:
Not blocking, but personally I'm not a fan of headings which are links in documentation.
Not something I see when looking at other Open Source projects.

Reply:
Good point, the left sidebar in Backstage serves this purpose as well. I'll update.